lib/cucumber_factory/factory.rb

```diff
@@ -100,8 +117,33 @@ def parse_named_creation(world, double_quote_name, single_quote_name, raw_model,
       end
 
       def parse_creation(world, raw_model, raw_variant, raw_attributes, raw_boolean_attributes, raw_multiline_attributes = nil)
-        build_strategy, transient_attributes = BuildStrategy.from_prose(raw_model, raw_variant)
+        build_strategy, transient_attributes = CucumberFactory::BuildStrategy.from_prose(raw_model, raw_variant)
         model_class = build_strategy.model_class
+        attributes = parse_attributes(world, model_class, raw_attributes, raw_boolean_attributes, raw_multiline_attributes, transient_attributes)
+        record = build_strategy.create_record(attributes)
+        remember_record_names(world, record, attributes)
+        record
+      end
+
+      def parse_update(world, raw_model, raw_name, raw_attributes, raw_boolean_attributes, raw_multiline_attributes = nil)
+        model_class = CucumberFactory::BuildStrategy.parse_model_class(raw_model)
+        attributes = parse_attributes(world, model_class, raw_attributes, raw_boolean_attributes, raw_multiline_attributes)
+        record = resolve_associated_value(world, model_class, model_class, model_class, raw_name)
+
+        if model_class.respond_to?(:create!) # This is an ActiveRecord class
```

Comment:
Even though it's just a few lines here, I strongly suggest to extract this logic into a `Cucumber::UpdateStragety` analogue to the `BuildStrategy`. I'd expect to see similar methods like the `active_record_strategy` etc.